linux-admin/part-2-tasks/tasks_othman.md

```diff
@@ -0,0 +1,154 @@
+# First step on command line.
+1. Display your current directory.
+
+pwd
+
+2. Change to the /etc directory.
+
+cd /etc
+
+3. Now change to your home directory using only three key presses.
+
+cd
+
+4. Go to the parent directory of the current directory.
+
+cd ..
+
+5. Go to the root directory.
+
+cd /
+
+6. List a long listing of the root directory using two ways.
+
+ll 
+ls -l
+
+7. Using man pages list all files under /usr/bin and show list of files in long listing format,human readable size format,and show the oldest files first.
+
+ls -lhtr
+
+8. Stay where you are, and list the contents of /bin and /sbin.
+
+ll /bin
+ll /sbin
+
+9. List the files in /boot in a human readable format.
+
+ll /boot -h
+
+10. Create in one command the directories ~/dir1/dir2/dir3 (dir3 is a subdirectory
+from dir2, and dir2 is a subdirectory from dir1 ).
+
+mkdir ~/dir1/dir2/dir3 -p
+
+11. create /tmp/test-fld folder and /tmp/test-File file and move /tmp/test-File inside /tmp/test-fld. Now remove /tmp/test-fld using two ways.
+
+mkdir /tmp/test-fld 
+touch /tmp/test-File
+mv /tmp/test-File /tmp/test-fld
+
+rmdir /tmp/test-fld --ignore-fail-on-non-empty
+rm /tmp/test-fld -d
+
+12. Display the type of file of /bin/cat, /etc/passwd and /usr/bin/passwd.
+
+file /bin/cat
+file /etc/passwd
+file /usr/bin/passwd
+
+13. Display the type of the file /usr/share/man/man1/bc.1.gz
+
+file /usr/share/man/man1/bc.1.gz
+
+14. Create the files today.txt and yesterday.txt by using touch command.
+
+touch today.txt yesterday.txt
+
+15. Change the date on yesterday.txt to match yesterday’s date.
+
+touch -d "yesterday" yesterday.txt
+
+16. Display the first 12 lines of /etc/services.
+
+head /etc/services -12
+
+17. Display the last line of /etc/passwd.
+
+tail /etc/passwd -1
+
+18. Use cat to create a file named count.txt that looks like this:
+One
+Two
+Three
+Four
+Five
+
+cat >count.txt
+One
+Two
+Three
+Four
+Five
+ctrl+d
+
+
+19. Use cp to make a backup of count.txt file to cnt.txt.
+
+cp count.txt cnt.txt
+
+20. Use cat to make a backup of count.txt file to catcnt.txt.
+
+cat count.txt > catcnt.txt
+
+21. Display catcnt.txt, but with all lines in reverse order (the last line first).
+
+tac catcnt.txt
+
+22. Use cat to create a file named tailing.txt that contains the contents of /etc/profile followed by the contents of /etc/passwd.
+
+cat /etc/profile /etc/passwd > tailing.txt
+
+23. Use more to display /var/log/messages.
+
+more /var/log/messages
+
+24. Display the readable character strings from the /usr/bin/passwd.
+
+strings /usr/bin/passwd
+
+25. Use ls to find the biggest file in /etc.
+
+ls -S /etc
```

Comment:
this the correct option but it will list all the files based on the sizes but you need to use head to filter the result and get the biggest file

`bash ls -S /etc |head -1`